app/javascript/components/ActionItemCards/index.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ActionItemCard.css';
```

Comment:
Is there a reason it's css first instead of just a styles.js file?

Reply:
I couldn't make the stlys.js work so I gave up